src/main/java/Duke.java

```diff
@@ -14,106 +9,25 @@ public static void main(String[] args) {
         System.out.println("Hello from\n" + logo);
         System.out.println("Konnichiwa! I am Duke the cat! What can I do for you? meow~ (^.___.^)");
```

Comment:
can try putting your System.out.println as a function under UI, so that the codes here will be neater?